src/Apps/Legion/Components/WizardShell/Internal/ModifyStep/Internal/EntitiesTab/EntitiesTab.styles.ts

```diff
@@ -1,26 +1,29 @@
-import { IProcessedStyleSet, mergeStyleSets } from '@fluentui/react';
-import { IEntitiesTabStyles } from './EntitiesTab.types';
-import { CONTENT_HEIGHT } from '../../../../WizardShell.styles';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import { TEntitiesTabStyles } from './EntitiesTab.types';
 
-export const getStyles: IProcessedStyleSet<IEntitiesTabStyles> = mergeStyleSets(
-    {
-        root: {
-            maxHeight: CONTENT_HEIGHT - 40, // Height - pivot
-            overflowY: 'auto'
-        },
-        columnWrapper: {
-            display: 'flex',
-            alignItems: 'center',
-            height: '100%'
-        },
-        idColumn: {
-            overflow: 'hidden',
-            textOverflow: 'ellipsis',
-            whiteSpace: 'nowrap'
-        },
-        newEntityIcon: {
-            minWidth: 10,
-            margin: '-16px 0 0 2px'
-        }
+export const EntityTabCSSVar = '--legion-entity-root-max-height';
+
+export const useEntitiesTabClassNames = makeStyles<TEntitiesTabStyles>({
+    root: {
+        maxHeight: `var(${EntityTabCSSVar})`,
```

Comment:
is there a reason why we used a css var here and defined the value of that var in the component with style? couldn't we keep it as is?

Reply:
also build test fails, can you check it out please?

Reply:
The height is a calculated value, based on an exported variable. I tried keeping it as is, but it seems `makeStyles` does not allow us to have calculated values as part of its CSS rules. I can try to calculate the value outside of the function. I'll let you know the result of that.

On another note: build is failing because of a snapshot, fixing it.

Reply:
Looks like `makeStyles` is gets its values on build time. Variables and calculated values need to be assigned to CSS vars, otherwise the build will fail.

Reply:
oh got it!